### PR TITLE
Version 4.0

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

File: app/models/changelog_data.php [hunk 1]
```diff
@@ -46,9 +47,119 @@
     '3.8.1' => '2015-06-24',
     '3.9'   => '2015-10-08',
     '3.10'  => '2015-11-13',
+    '4.0'   => '2016-01-04',
 ];
 
 $changelog = [
+    '4.0' => [
+        [
+            'message' => ['Propose search suggestions as your type in the search field'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: your->you

File: app/models/changelog_data.php [hunk 2]
```diff
@@ -46,9 +47,119 @@
...
+        ],
+        [
+            'message' => ['Stop hard-coding version information in the application.'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'commit'  => ['b7dbe66c4837ef557a40c57ee5baab86dff2d353'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Update Travis CI configuration toexclude xdebug and add PHP 5.6 and 7 to the testing matrix'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: to exclude

File: app/models/changelog_data.php [hunk 3]
```diff
@@ -46,9 +47,119 @@
...
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Upgraded OS from Debian 7 to 8. Transvision minimum PHP requirement moved from 5.4 to 5.6'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'authors' => ['geb'],
+        ],
+        [
+            'message' => ['Host OpenSans font ourselves to improve page load tome instead of loading it from mozilla.org'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: time

File: app/models/changelog_data.php [hunk 4]
```diff
@@ -46,9 +47,119 @@
...
+    '4.0' => [
+        [
+            'message' => ['Propose search suggestions as your type in the search field'],
+            'section' => ['end_user'],
+            'type'    => ['new'],
+            'issues'  => [551],
+            'authors' => ['pascal', 'flod'],
+        ],
+        [
+            'message' => ['Create a <a href="/rss/">RSS</a> feed for release notes'],
```
Comment (TheoChevalier) [inline app/models/changelog_data.php]: I think we need a full link for external RSS readers

File: app/models/changelog_data.php [hunk 5]
```diff
@@ -46,9 +47,119 @@
...
+        ],
+        [
+            'message' => ['Update results count when filtering search results by component'],
+            'section' => ['end_user'],
+            'type'    => ['bug'],
+            'commit'  => ['f26b5fb58047ba8c460ff6bd41dabb461b18a91d'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Stop hard-coding version information in the application.'],
```
Comment (flodolo) [inline app/models/changelog_data.php]: Remove final period for consistency with other items.

File: app/models/changelog_data.php (hunk 4 above)
Comment (flodolo) [inline app/models/changelog_data.php]: Why? It works fine for me

File: app/models/changelog_data.php [hunk 6]
```diff
@@ -46,9 +47,119 @@
...
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Transvision moved to a faster server with hardware redundancy sponsored by <a href="https://www.gandi.net/">Gandi</a>'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Upgraded OS from Debian 7 to 8. Transvision minimum PHP requirement moved from 5.4 to 5.6'],
```
Comment (TheoChevalier) [inline app/models/changelog_data.php]: I see you updated the Wiki, but we will need to update the readme file as well :) https://github.com/mozfr/transvision#dependencies

File: app/models/changelog_data.php (hunk 4 above)
Comment (TheoChevalier) [inline app/models/changelog_data.php]: If you read it from Thunderbird, I think it won’t work (isn’t it the reason why we updated all the relative links in this file?). If it works then we can probably revert the change we made

File: app/models/changelog_data.php (hunk 6 above)
Comment (TheoChevalier) [inline app/models/changelog_data.php]: Didn’t see it’s done in this PR :)

File: app/models/changelog_data.php (hunk 4 above)
Comment (pascalchevrel) [inline app/models/changelog_data.php]: I just tried and it works without the domain.

Comment (TheoChevalier) [inline app/models/changelog_data.php]: Ok sweet, so we can s/https:\/\/transvision.mozfr.org// (Or I can do a separate PR if you want)
